src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageDependencies.cs

```diff
@@ -361,6 +361,11 @@ private void GetFileDependencies(LockFileTargetLibrary package, string targetNam
                     item.SetMetadata(MetadataKeys.ParentTarget, targetName); // Foreign Key
                     item.SetMetadata(MetadataKeys.ParentPackage, packageId); // Foreign Key
 
+                    if (fileGroup == FileGroup.FrameworkAssembly)
```

Comment:
I think it would be helpful to have a comment here explaining how the FrameworkAssembly reference is expressed in the lockfile.  From the code I gather that it has a path which instead of referring to a file in the package is just the name of the framework reference to add.